DxDispatch/src/model/Model.cpp

```diff
@@ -19,19 +19,27 @@ Model::Model(
     for (auto& dispatchableDesc : m_dispatchableDescs) 
     { 
         m_dispatchableDescsByName[dispatchableDesc.name] = &dispatchableDesc;
-
-        if (std::holds_alternative<DmlDispatchableDesc>(dispatchableDesc.value))
+        
+        if (std::holds_alternative<DmlDispatchableDesc>(dispatchableDesc.value) ||
+            std::holds_alternative<DmlSerializedGraphDispatchableDesc>(dispatchableDesc.value))
         {
-            auto& dmlDispatchableDesc = std::get<DmlDispatchableDesc>(dispatchableDesc.value);
-            for (auto& binding : dmlDispatchableDesc.initBindings)
+            const auto& initBindings = std::holds_alternative<DmlDispatchableDesc>(dispatchableDesc.value)
```

Comment:
I think another approach to re-use the binding verification logic is to move the verification logic into a method and use it. So that it can be used in future as well.

Reply:
done